packages/@okta/vuepress-site/docs/guides/event-hook-ngrok/main/index.md

```diff
@@ -0,0 +1,159 @@
+
+1. In the Admin Console, go to **Workflow** > **Event Hooks**.
+2. Locate the Event Hook you created during the set-up step. In this example, select `Deactivated User Event Hook` or the name you gave the Event Hook.
+3. Click the **Actions** menu for this hook, and select **Preview**.
+4. In the **Configure Event Hook request** section, select an event from the **Event Type** drop-down menu. In this example, there is only one: `User deactivated (user.lifecycle.deactivate)`.
+5. Select a previous recent event (in this case, a user deactivation) from the **System Log Event** drop-down menu. The **Preview & Deliver Event Hook** section populates the JSON body of the Event Hook. If no event is available, the JSON body populates with sample data.
+6. Optionally, click **Edit** to modify the JSON body call. For example, you can change the `target` object's property, `displayName`, to `John Doe`.
+7. Ensure that both your ngrok session and local sample application are running.
+
+    >**Note:** If you start a new ngrok session at any time, make sure to update the Event Hook URL.
+
+8. Click **Deliver Request**. The Event Hook Preview displays the status request as either successful or a failure. Check your local application console. The following message displays if successful:
+
+     `The user John Doe has been deactivated on the Okta org!`
+
+9. Check your ngrok terminal session. Each call recorded by ngrok appears in the terminal.
+
+    ![A screen shot of the ngrok terminal that includes a simple line item of a call.](/img/ngrok-and-event-hooks-terminal-calls.png)
+
+### Test
+
+To run a test of your Event Hook:
+
+1. Ensure that both your ngrok session and local sample application are running.
+2. In your Okta org, sign in as an administrator and create a test user in the Admin Console.
+    - Go to **Directory** > **People**, and click **Add Person**.
+    - As an example, add the user John Doe as seen from the [Event Object sample code](/docs/guides/event-hook-implementation/event-object).
+3. For this user, select the User's profile by clicking John Doe's name.
+4. Click the **More Actions** drop-down menu, and select **Deactivate**.
+5. Confirm the deactivation.
+6. Navigate back to your local application's console. You should see the following output to the console:
+
+    `The user John Doe has been deactivated on the Okta org!`
+7. Check your ngrok terminal session. Each call recorded by ngrok appears in the terminal.
+
+    ![A screen shot of the ngrok terminal that includes a simple line item of a call.](/img/ngrok-and-event-hooks-terminal-calls.png)
+
+### Review ngrok dashboard
+
+The ngrok inspection interface provides an opportunity to review all calls to your local application. See [ngrok documentation](https://ngrok.com/docs#inspect-requests) for details on using this interface. With the ngrok utility running, open the following URL in a browser: `http://localhot:4040`.
```

Comment:
I think there's a typo here - it should be "http://localhost:4040/"

Reply:
Fixed.